selinux/tabrmd.te

```diff
@@ -15,12 +15,8 @@ dev_rw_tpm(tabrmd_t)
 logging_send_syslog_msg(tabrmd_t)
 
 optional_policy(`
+    dbus_stub()
     dbus_system_domain(tabrmd_t, tabrmd_exec_t)
+    allow system_dbusd_t tabrmd_t:unix_stream_socket rw_stream_socket_perms;
 ')
 
```

Comment:
I tested this and it works for me, thanks a lot @wrabcak!

Just one small comment, there are two typos in the commit message:

s/useed/used
s/distrubytion/distribution

Reply:
I fixed typos. Sorry for that.